frontend/README.md

```diff
@@ -0,0 +1,93 @@
+# Abstract Account App
+
+## Introduction
+
+The Abstract Account App is a Next.js-based application that interacts with the
+Abstract Account system on the blockchain. It enables users to connect their
+wallets, create and manage Abstract Accounts, query account information, and
+interact with smart contracts in a type-safe manner using Abstract Codegen.
+
```

Comment:
We should include a techstack connection 
react
typescript 
Next.js 
shadcn
tailwind

graz
react-query
abstract.js


with links to their documentation.